daemon/api.go

```diff
@@ -116,124 +116,136 @@ var api = []*Command{
 
 var servicestateControl = servicestate.Control
 
+const (
+	polkitActionLogin            = "io.snapcraft.snapd.login"
+	polkitActionManage           = "io.snapcraft.snapd.manage"
+	polkitActionManageInterfaces = "io.snapcraft.snapd.manage-interfaces"
+)
+
 var (
-	// see daemon.go:canAccess for details how the access is controlled
+	// see access.go for details how the access is controlled
 	rootCmd = &Command{
-		Path:    "/",
-		GuestOK: true,
-		GET:     tbd,
+		Path:       "/",
+		ReadAccess: openAccess{},
+		GET:        tbd,
 	}
 
 	sysInfoCmd = &Command{
-		Path:    "/v2/system-info",
-		GuestOK: true,
-		GET:     sysInfo,
+		Path:       "/v2/system-info",
+		ReadAccess: openAccess{},
```

Comment:
I think the *Access should go after the GET/POST fields, like in the definition of the struct